DribbbleSDKExample/DribbbleSDKExample/MainViewController.m

```diff
@@ -37,7 +38,7 @@
 @interface MainViewController () <UITableViewDelegate, UITableViewDataSource>
 
 @property (strong, nonatomic) DRApiClient *apiClient;
-
+@property (strong, nonatomic) AppDelegate *delegate;
```

Comment:
remove data model properties from delegate and store them in this view controller